tmt/steps/provision/__init__.py

```diff
@@ -1112,8 +1112,9 @@ def requires(cls) -> list['tmt.base.Dependency']:
     def acquire_log(self, log_name: str) -> Optional[str]:
         """
         Fetch and return content of a log.
+
         :param log_name: name of the log.
-        :returns: content of the log.
+        :returns: content of the log, or None if the log cannot be retrieved.
```

Comment:
"\`\`None\`\`" instead of "None", to make it clear it's not just any "none", but literally Python's `None` object.